src/main/java/com/mraof/minestuck/inventory/musicplayer/IMusicPlaying.java

```diff
@@ -0,0 +1,15 @@
+package com.mraof.minestuck.inventory.musicplayer;
+
+import com.mraof.minestuck.block.EnumCassetteType;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.world.item.ItemStack;
+
+public interface IMusicPlaying {
+	void setCassetteType(EnumCassetteType cassetteType);
+	
+	void setCurrentMusicPlayer(ItemStack itemStack);
```

Comment:
The two setters always seem to be used together. Would it make sense to combine them?